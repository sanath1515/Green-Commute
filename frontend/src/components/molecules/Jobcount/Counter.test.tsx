import React from "react";
import Counter from "./Count";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";

describe("Counter renders correctly", () => {
  test("should match snapshot", () => {
    const { asFragment } = render(<Counter number="1000" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
