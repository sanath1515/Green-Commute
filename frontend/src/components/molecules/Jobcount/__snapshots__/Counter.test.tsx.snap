// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter renders correctly should match snapshot 1`] = `
<DocumentFragment>
  <h2
    class="MuiTypography-root makeStyles-text-1 MuiTypography-h2"
  >
    1000
  </h2>
  <h3
    class="MuiTypography-root makeStyles-text1-2 MuiTypography-h3"
  />
</DocumentFragment>
`;
